pallets/credit/src/lib.rs

```diff
@@ -379,22 +374,25 @@ pub mod pallet {
             weight
         }
 
-        fn init_credit_history(account_id: &T::AccountId, credit_data: CreditData) -> Weight {
-            let mut weight = T::DbWeight::get().reads_writes(1, 0);
-            match T::NodeInterface::get_onboard_time(account_id) {
-                Some(block) => {
-                    let onboard_era = Self::block_to_era(block);
-                    UserCreditHistory::<T>::insert(account_id, vec![(onboard_era, credit_data)]);
-                    weight = weight.saturating_add(T::DbWeight::get().reads_writes(0, 1));
-                }
-                None => (),
+        fn init_credit_history(
```

Comment:
Searched `init_credit_history`, didn't found where `era = None` is passed in , so I think you can just use a `u32` here.

Reply:
ok